tests/api/routers/test_institution_cleanup.py

```diff
@@ -45,28 +44,32 @@ def test_institution_delete_helper(app_fixture: FastAPI, mocker: MockerFixture,
     delete_group_mock.assert_called_once_with("123456E2ETESTBANK123")
 
     # Delete Domains Fail
-    delete_domains_mock.return_value = None
-    delete_helper("123456E2ETESTBANK123", session_mock)
-    assert "Domain(s) for LEI 123456E2ETESTBANK123 not deleted." in caplog.text
+    delete_domains_mock.side_effect = IOError("Test")
+    with pytest.raises(Exception) as e:
+        delete_helper("123456E2ETESTBANK123", session_mock)
+    assert isinstance(e.value, RegTechHttpException)
+    assert e.value.name == "Domains Delete Failed"
 
     # Delete SBL Type Fail
-    delete_domains_mock.return_value = ok_res
-    delete_sbl_type_mock.return_value = None
-    delete_helper("123456E2ETESTBANK123", session_mock)
-    assert "sbl type(s) for LEI 123456E2ETESTBANK123 not deleted." in caplog.text
+    delete_domains_mock.side_effect = None
+    delete_sbl_type_mock.side_effect = IOError("Test")
+    with pytest.raises(Exception) as e:
+        delete_helper("123456E2ETESTBANK123", session_mock)
+    assert isinstance(e.value, RegTechHttpException)
+    assert e.value.name == "Sbl Type Delete Failed"
 
     # Delete Institution Fail
-    delete_sbl_type_mock.return_value = ok_res
+    delete_sbl_type_mock.side_effect = None
     delete_institution_mock.return_value = None
     with pytest.raises(Exception) as e:
         delete_helper("123456E2ETESTBANK123", session_mock)
     assert isinstance(e.value, RegTechHttpException)
-    assert e.value.name == "Institution to be deleted Not Found"
+    assert e.value.name == "Institution Delete Failed"
```

Comment:
It's a good idea here to test the details, since name is a static string but the details are meant to convey what LEI failed and there's actual substitution/formatting going on.  So to verify the details capture the correct LEI.  So capturing e.value.name and e.value.details have the expected results is a more thorough test of what will be sent to the frontend.

Reply:
Added details check to tests.